### ISSUE TITLE
Metadata.xml is in another zone

### ISSUE THREAD

**Issue by petescarth** [cirrus-geo/cirrus-earth-search#8]
When using the S2 L2A data for biophysical analysis it is useful to correct for BRDF effects to allow time series comparisons and for seamless mosaicing e.g. [Flood, Neil (2020)](https://doi.org/10.6084/m9.figshare.12820268.v1). This can be done in various ways, including on the fly using AWS lambda. This correction requires the solar and sensor geometry grids contained in the original metadata.xml. The link to the original metadata.xml on roda.sentinel-hub.com is captured in the STAC json and it can also be pulled from the sentinel-s2-l2a bucket but this is inconvenient in a production environment for a number of reasons:

- Neither of these endpoints are directly accessible from lambda running in a us-central-2 VPC next to the COGs without setting up a NAT gateway
- The load and latency on roda.sentinel-hub.com from, say, ~1000 lambda instances simultaneously accessing this endpoint is significant if running outside a VPC
- The latency (and hence cost) when accessing metadata from S2 in the eu-central zone from us-central is also significant

Given the importance of the geometry grids in meeting the CEOS CARD4L specification, it would seem sensible having them colocated with the image data in the sentinel-s2-l2a-cogs bucket. 

This issue is to request that the workflow for the COG production include colocated metadata.xml

**Comment by matthewhanson** [comment]
@petescarth Apologies for the late reply, feel like I'm still catching up from PTO at year's end.

What you say makes perfect sense.  When I originally did this I figured "why duplicate data" since we aren't changing it, let's just point to the original.  But all your points are valid reasons for not just metadata.xml, but all the assets being in one location rather than having to go EU for some of them.  Also nice to be able to sync a whole directory from s3 and have it contain all the assets.

I can make this change for data going forward easily enough, but going back and adding it to the existing archive is going to take longer. I've got misc other metadata updates to make to the archive, so would like to make one pass through to make all the updates at once. I'll add this to the list, and will keep this issue open in the meantime.
